aws-for-fluent-bit.yaml

```diff
@@ -0,0 +1,86 @@
+package:
+  name: aws-for-fluent-bit
+  version: 2.31.12.20230629
+  epoch: 0
+  description: AWS provides a Fluent Bit image with plugins for both CloudWatch Logs and Kinesis Data Firehose.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+      - fluent-bit
+      - go
```

Comment:
Why is `go` a runtime dependency?  That would result in `go` being in the generated image, for example.

Reply:
I think we can just drop `go` in general here.

Reply:
Ahhh I was copied from your review. 🙈 https://github.com/wolfi-dev/os/pull/3473#discussion_r1259931643 Let me remove.